tests/integration/relations/pgbouncer_provider/test_pgbouncer_provider.py

```diff
@@ -309,31 +226,28 @@ async def test_scaling(ops_test: OpsTest):
     )
 
 
+@pytest.mark.unstable
 async def test_relation_broken(ops_test: OpsTest):
```

Comment:
Also disabled in the pgbouncer charm (user is not deleted once the last unit goes down).